### DISCUSSION TITLE
Build and deploy failed.

### DISCUSSION THREAD

**Discussion by mishba-ai** [microsoft/SATechnicalOnboarding#103] [Questions]
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/121809535/d6e50e8b-a243-4a1d-b219-2807336d8706)
 Don't know what's the issue.
 anyone could help me to fix this?

**Comment by AnthonyByansi** [comment]
Hi @mishba-ai I trust this issue has been resolved on teams. All you need to pay attention to is the deployment details for your app.
![Change Build Presite](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/6927f5b2-393e-465f-a6c6-06c987f4c665)

**Comment by mishba-ai** [comment]
Hello, I have made the changes as per your instructions but I am still experiencing some issues.
It appears that the CI/CD workflow has failed for the main branch. 
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/121809535/c8602368-f05a-49f5-ad33-9aec3add8a97)

**Reply by AnthonyByansi** [reply]
Hi @mishba-ai please note that the output location has to be `dist` . It would be much easier if you do it  directly from the `Azure portal` instead by help of Vs code extension. 
![301968770-0f067bff-607c-4186-b36b-834bbad46d97](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/19c90c80-5c0a-45bf-8d14-1c0d2d39dbab)
